subprojects/docs/src/docs/userguide/plugins.adoc

```diff
@@ -193,13 +193,41 @@ BUILD SUCCEEDED
 The `pluginManagement {}` DSL is currently <<feature_lifecycle,incubating>>. Please be aware that the DSL and other configuration may change in later Gradle versions.
 ====
 
+The `pluginManagement {}` block may only appear in either the `settings.gradle` file, where it must be the first block in the file, or in an <<init_scripts,Initialization Script>>.
+
+*settings.gradle*
+[source,groovy]
+----
+pluginManagement {
+  resolutionStrategy {
+    // ...
+  }
+  repositories {
+    // ...
+  }
+}
+----
+
+*~/.gradle/init.gradle*
+[source,groovy]
+----
+settingsEvaluated { settings ->
+  settings.pluginManagement.resolutionStrategy {
```

Comment:
Can you change this to just one block like the example above? That would look just a bit more idiomatic. :)